### DISCUSSION TITLE
Education pack not provided yet

### DISCUSSION THREAD

**Discussion by XploitGh0st** [community/community#131065] [GitHub Education] [answered]
### Hi everyone,

I have applied for github student pack. Is been three days till I don't receive a mail for confirmation. Pls make the required steps to deliver me soon with it.

**Comment by RajeevRanjan5642** [comment, marked as answer]
Hey👋

I think I can help you with that because I was also in the same boat. I had applied on 29th of June and received mail of confirmation on 2nd of July. They take 4-5 days for processing the request since the appliers are in great number. So, you should just wait for 1-2 days and you will definitely receive a mail from their side 💌.

Hope it helps 💖!!

**Reply by XploitGh0st** [reply]
Thank You so  much for your help!

**Comment by qyzqq** [comment]
How the f%ck i got accepted then rejected after 2 days
